coffee/chosen.jquery.coffee

```diff
@@ -358,7 +358,7 @@ class Chosen extends AbstractChosen
       else
         this.single_set_selected_text(this.choice_label(item))
 
-      this.results_hide() unless (evt.metaKey or evt.ctrlKey) and @is_multiple
+      this.results_hide() unless (@autohide_results_multiple or evt.metaKey or evt.ctrlKey) and @is_multiple
```

Comment:
This condition is backwards, right?  If the parenthetical is true, that means _don't_ hide, but the first condition being true means _do_ hide.